packages/currency/src/currencyManager.ts

```diff
@@ -36,27 +36,40 @@ export class CurrencyManager<TMeta = unknown> implements ICurrencyManager<TMeta>
   }
 
   /**
-   * Gets a supported currency from a symbol, symbol-network or address.
+   * Gets a supported currency from a symbol, symbol-network, currency definition id or address.
```

Comment:
Why not having both: from id  & from symbol?

something like:  `from(idOrSymbolOrAddress: string | undefined)` 
Searching first from id then Symbol or address ?

Reply:
Yup, we still search by the symbol. The parameter name is more generic because I think it's ugly to have 2 `or`.

Reply:
The id is in the last order because I put the id in the second search but it breaks the test for conflicting currency.

Reply:
Oh sorry! I should have read the code first :sweat_smile:
I let @benjlevesque make the call for the naming change.